src/js/gsTabSuspendManager.js

```diff
@@ -303,6 +303,10 @@ var gsTabSuspendManager = (function() {
       let discardInPlaceOfSuspend = gsStorage.getOption(
         gsStorage.DISCARD_IN_PLACE_OF_SUSPEND,
       );
+      if (tab.cookieStoreId != "firefox-default") {
```

Comment:
Could not make it work with FF tab containers, however, discarding works fine here